aws/ci_build

```diff
@@ -226,11 +225,15 @@ def mark_dtt_green
 end
 
 # Log details of a build to metrics database.
-# @param [Symbol] environment - :success or :failed
+# TODO: (suresh) This method will eventually be used to write build status for all managed environments.
+# @param [Symbol] environment
 # @param [String] commit_hash
-# @param [String] status
+# @param [String] status - :start, :success, :failed
```

Comment:
(nit) looks like the expected type is `[Symbol]` not `[String]` here (though it doesn't seem to make a difference in the current implementation since it's only interpolated in a string)